### PR TITLE
upgrade vault to 0.9.6

### PR DESCRIPTION
**Pull request by xiujiao**

### CODE REVIEW

**Comment by jhunt** [comment]
Can you resolve the merge conflicts?

File: README.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ all necessary utilities for running BOSH deployments, including:
   - **genesis** - For creating multi-tiered deployment repos
   - **spruce** - A YAML multitool for managing BOSH manifests
   - **safe** - An alternate CLI for Hashicorp's Vault
+  - vault - The offical CLI for Hashicorp's Vault
```
Comment (jhunt) [inline README.md]: We specifically don't call this out in the README; `vault` is a dependency of `safe`, nothing more.

Comment (xiujiao) [reply README.md]: Removed :)

**Comment by jhunt** [comment]
There don't seem to be any changes to review now?

**Comment by xiujiao** [comment]
@jhunt you are right. the vault part was merged with the bosh PR already

**Pull request closed without merging**